### DISCUSSION TITLE
Lesson 4 : Connect to ganache

### DISCUSSION THREAD

**Discussion by wilonweb** [smartcontractkit/full-blockchain-solidity-course-py#883] [Q&A] [answered]
Hello 
For connecting to ganache the teacher use his own Ganache RPC server number 
`w3 = Web3(Web3.HTPProvider("http://0.0.0.0:8455"))`
On my ganache the RPC server number is 
`HTTP://127.0.0.1:7545`
So i have to put 
`w3 = Web3(Web3.HTPProvider("http://127.0.0.1:7545"))` ?

**Comment by cromewar** [comment, marked as answer]
Hello @NinjaTurtles-cloud, yes indeed the RPC server number should change independently on each computer.
Anyways 0.0.0.0 and 127.0.0.1 are the same as both represents `localhost`, but yes you should use the one your terminal es giving you.
